insights/specs/default.py

```diff
@@ -402,6 +402,7 @@ class DefaultSpecs(Specs):
     mount = simple_command("/bin/mount")
     mountinfo = simple_file("/proc/self/mountinfo")
     mounts = simple_file("/proc/mounts")
+    mpirun_version = simple_command("/usr/local/bin/mpirun --version")
```

Comment:
Is this the expected location of this command?  Are there other typical locations where this could be installed?  In the parser docstring you specify `/usr/bin/mpirun`.  If it may be located in either location then you can use the `first_of` spec, like [this](https://github.com/RedHatInsights/insights-core/blob/master/insights/specs/default.py#L367).

Reply:
This makes sense to me because recently I noticed that the generic openmpi package install the binary in ` /usr/lib64/openmpi/bin/mpirun`

Reply:
@psachin if we will use the `usr/lib64/openmpi/bin/mpirun` binary to get the version, the out we get is totally different i.e. mpirun (Open MPI) 4.1.1
from 4.1.1 we can't detect the year as we don't have correct mapping. so i think we can enhance the rule later, but it is currently sufficient for the jira i'm working on.

Reply:
Ok @mohitkumarrh I think we can merge the changes for now.